### ISSUE TITLE
image resolution dependent bass clef shift

### ISSUE THREAD

**Issue by KannebTo** [Audiveris/audiveris#394]
This image gets correctly detected and exported (screenshot from MuseScore):
![example1](https://user-images.githubusercontent.com/24896645/84314438-550bb300-ab68-11ea-89c9-039ad73a0ec3.png)

The same content in slightly reduced resolution:
![example2](https://user-images.githubusercontent.com/24896645/84314439-55a44980-ab68-11ea-8507-7c966767e751.png)
Strangely the re-import from this one in MuseScore (.mxl) results in:
![result-example2](https://user-images.githubusercontent.com/24896645/84314440-563ce000-ab68-11ea-82ff-08055e1ae1fd.png)

In Audiveris Data tab it looks the same as the first one.

Audiveris version: 86d866b0d

**Comment by hbitteur** [comment]
@KannebTo 
A different resolution results in different pixels processed by OMR, hence perhaps different results.
What were the resolutions for case 1 and case 2?
Could you post both images so that we can reproduce the problem?

**Comment by KannebTo** [comment]
@hbitteur
You can't see the images? I'm looking at this issue through the github [web interface](https://github.com/Audiveris/audiveris/issues/394) and the images are there. Are you using some other tool or email only?

Here are the references:
* [case 1](https://user-images.githubusercontent.com/24896645/84314438-550bb300-ab68-11ea-89c9-039ad73a0ec3.png) (working) 1120 x 424 px
* [case 2](https://user-images.githubusercontent.com/24896645/84314439-55a44980-ab68-11ea-8507-7c966767e751.png) 917 x 331 px
  * [result of case 2](https://user-images.githubusercontent.com/24896645/84314440-563ce000-ab68-11ea-82ff-08055e1ae1fd.png) (after MuseScore import)

**Comment by hbitteur** [comment]
My mistake. The first image is annotated with "(screenshot from MuseScore)", so I thought the image was a screen-copy of MuseScore.

I processed your 2 images, the exported XML content is different:
First:
```xml
        <clef>
          <sign>F</sign>
          <line>4</line>
        </clef>
```
Second:
```xml
        <clef>
          <sign>F</sign>
          <line>3</line>
        </clef>
```
I now have to find out what this comes from.
Stay tuned...

**Comment by KannebTo** [comment]
Yes, it is a screen-copy of MuseScore and an input image.
At first i noticed this bug on a real-world score and than i produced these input images with a short example using MuseScore to find out under what conditions this happens.

The fact that this is resolution dependent points to the OMR. But when i print the transcribed book directly to a PDF, every element is in the right place. So it was actually correctly recognized?

Thanks for working on that.

**Comment by hbitteur** [comment]
This was just the tip of an iceberg.
In fact the "line" element was wrong for G clef as well but had never been detected.
This was due to wrong pitch computation for certain entities.

And changing image resolution had the side effect of rounding a (wrong) double value from one integer value to the previous one. Which revealed the problem!

Audiveris PDF output does not use the pitch, since it knows the precise location of any entity. Hence the wrong pitch/line value had no visible effect.
On the opposite, MuseScore obviously uses clef line information to drive its layout.

So, I decided to unify pitch computing for alters, keys and clefs to take potential vertical non-symmetry into account.

This is available in latest commit 9c7e479f8e2df7c1ad3b5e81703bf6dbe280d681
